client/ayon_core/hosts/maya/api/alembic.py

```diff
@@ -0,0 +1,306 @@
+            objects are written to the Alembic file. Only the current UV map is
+            included.
+
+        writeColorSets (bool): Write all color sets on MFnMeshes as
+            color 3 or color 4 indexed geometry parameters with face varying
+            scope. Defaults to False.
+
+        writeFaceSets (bool): Write all Face sets on MFnMeshes.
+            Defaults to False.
+
+        wholeFrameGeo (bool): Data for geometry will only be written
+            out on whole frames. Defaults to False.
+
+        worldSpace (bool): When on, the top node in the node hierarchy is
+            stored as world space. By default, these nodes are stored as local
+            space. Defaults to False.
+
+        writeVisibility (bool): Visibility state will be stored in
+            the Alembic file.  Otherwise everything written out is treated as
+            visible. Defaults to False.
+
+        writeUVSets (bool): Write all uv sets on MFnMeshes as vector
+            2 indexed geometry parameters with face varying scope. Defaults to
+            False.
+
+        writeCreases (bool): If the mesh has crease edges or crease
+            vertices, the mesh (OPolyMesh) would now be written out as an OSubD
+            and crease info will be stored in the Alembic file. Otherwise,
+            creases info won't be preserved in Alembic file unless a custom
+            Boolean attribute SubDivisionMesh has been added to mesh node and
+            its value is true. Defaults to False.
+
+        dataFormat (str): The data format to use for the cache,
+                          defaults to "ogawa"
+
+        step (float): The time interval (expressed in frames) at
+            which the frame range is sampled. Additional samples around each
+            frame can be specified with -frs. Defaults to 1.0.
+
+        attr (list of str, optional): A specific geometric attribute to write
+            out. Defaults to [].
+
+        attrPrefix (list of str, optional): Prefix filter for determining which
+            geometric attributes to write out. Defaults to ["ABC_"].
+
+        root (list of str): Maya dag path which will be parented to
+            the root of the Alembic file. Defaults to [], which means the
+            entire scene will be written out.
+
+        stripNamespaces (bool): When on, any namespaces associated with the
+            exported objects are removed from the Alembic file. For example, an
+            object with the namespace taco:foo:bar appears as bar in the
+            Alembic file.
+
+        verbose (bool): When on, outputs frame number information to the
+            Script Editor or output window during extraction.
+
+        preRollStartFrame (float): The frame to start scene
+            evaluation at.  This is used to set the starting frame for time
+            dependent translations and can be used to evaluate run-up that
+            isn't actually translated. Defaults to 0.
```

Comment:
Should we care about the fact that the arguments of the function are sorted different than the arguments shown in the docstring?

```suggestion
    file,
    attr=None,
    attrPrefix=None,
    dataFormat="ogawa",
    endFrame=None,
    eulerFilter=True,
    frameRange="",
    noNormals=False,
    preRoll=False,
    preRollStartFrame=0,
    renderableOnly=False,
    root=None,
    selection=True,
    startFrame=None,
    step=1.0,
    stripNamespaces=True,
    uvWrite=True,
    verbose=False,
    wholeFrameGeo=False,
    worldSpace=False,
    writeColorSets=False,
    writeCreases=False,
    writeNormals=False,
    writeFaceSets=False,
    writeUVSets=False,
    writeVisibility=False
):
    """Extract a single Alembic Cache.

    This extracts an Alembic cache using the `-selection` flag to minimize
    the extracted content to solely what was Collected into the instance.

    Arguments:
        file (str): The filepath to write the alembic file to.

        attr (list of str, optional): A specific geometric attribute to write
            out. Defaults to [].

        attrPrefix (list of str, optional): Prefix filter for determining which
            geometric attributes to write out. Defaults to ["ABC_"].

        dataFormat (str): The data format to use for the cache,
                          defaults to "ogawa"

        endFrame (float): End frame of output. Ignored if `frameRange`
            provided.

        eulerFilter (bool): When on, X, Y, and Z rotation data is filtered with
            an Euler filter. Euler filtering helps resolve irregularities in
            rotations especially if X, Y, and Z rotations exceed 360 degrees.
            Defaults to True.

        frameRange (tuple or str): Two-tuple with start and end frame or a
            string formatted as: "startFrame endFrame". This argument
            overrides `startFrame` and `endFrame` arguments.

        noNormals (bool): When on, normal data from the original polygon
            objects is not included in the exported Alembic cache file.

        preRoll (bool): This frame range will not be sampled.
            Defaults to False.

        preRollStartFrame (float): The frame to start scene
            evaluation at.  This is used to set the starting frame for time
            dependent translations and can be used to evaluate run-up that
            isn't actually translated. Defaults to 0.

        renderableOnly (bool): When on, any non-renderable nodes or hierarchy,
            such as hidden objects, are not included in the Alembic file.
            Defaults to False.

        root (list of str): Maya dag path which will be parented to
            the root of the Alembic file. Defaults to [], which means the
            entire scene will be written out.

        selection (bool): Write out all all selected nodes from the
            active selection list that are descendents of the roots specified
            with -root. Defaults to False.

        startFrame (float): Start frame of output. Ignored if `frameRange`
            provided.

        step (float): The time interval (expressed in frames) at
            which the frame range is sampled. Additional samples around each
            frame can be specified with -frs. Defaults to 1.0.

        stripNamespaces (bool): When on, any namespaces associated with the
            exported objects are removed from the Alembic file. For example, an
            object with the namespace taco:foo:bar appears as bar in the
            Alembic file.

        uvWrite (bool): When on, UV data from polygon meshes and subdivision
            objects are written to the Alembic file. Only the current UV map is
            included.

        verbose (bool): When on, outputs frame number information to the
            Script Editor or output window during extraction.

        wholeFrameGeo (bool): Data for geometry will only be written
            out on whole frames. Defaults to False.

        worldSpace (bool): When on, the top node in the node hierarchy is
            stored as world space. By default, these nodes are stored as local
            space. Defaults to False.

        writeColorSets (bool): Write all color sets on MFnMeshes as
            color 3 or color 4 indexed geometry parameters with face varying
            scope. Defaults to False.

        writeCreases (bool): If the mesh has crease edges or crease
            vertices, the mesh (OPolyMesh) would now be written out as an OSubD
            and crease info will be stored in the Alembic file. Otherwise,
            creases info won't be preserved in Alembic file unless a custom
            Boolean attribute SubDivisionMesh has been added to mesh node and
            its value is true. Defaults to False.

        writeFaceSets (bool): Write all Face sets on MFnMeshes.
            Defaults to False.

        writeUVSets (bool): Write all uv sets on MFnMeshes as vector
            2 indexed geometry parameters with face varying scope. Defaults to
            False.

        writeVisibility (bool): Visibility state will be stored in
            the Alembic file.  Otherwise everything written out is treated as
            visible. Defaults to False.
```

Reply:
By the way, this also still takes both `noNormals` and `writeNormals`? One should be removed?

`writeNormals` still seems unused in the function and as such does nothing?

Reply:
`writeNormals` has been removed now.